src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int randomIndexColor = new Random().nextInt(Colors.values().length);
```

Comment:
should we create new instance of Random every time when the method is called?
create new var `Random` type  as class property and initialize there.